crates/sui/src/client_commands.rs

```diff
@@ -1909,7 +1872,6 @@ impl Display for SuiClientCommandResult {
             SuiClientCommandResult::ReplayTransaction => {}
             SuiClientCommandResult::ReplayBatch => {}
             SuiClientCommandResult::ReplayCheckpoints => {}
-            SuiClientCommandResult::NoOutput => {}
```

Comment:
Hmm, something went wrong, maybe a rebase will fix things?

Reply:
this is super weird some changes I did not make got added to this pr